deploy/helm/sumologic/conf/traces/traces.otelcol.conf.yaml

```diff
@@ -1,4 +1,7 @@
 {{- $yamlFile := toYaml $.Values.otelcol.config }}
-{{- $fullname := include "sumologic.fullname" . }}
-{{- $url := list "http://" $fullname ":9411/api/v2/spans" | join "" }}
-{{- $yamlFile | replace "exporters.zipkin.url_replace" $url | nindent 2 }}
+{{- $tracesEndpoint := include "sumologic.traces.endpoint" . | quote }}
+{{- $sourceName := include "sumologic.sourceName" . | quote }}
+{{- $sourceCategory := include "sumologic.sourceCategory" . | quote }}
+{{- $sourceCategoryPrefix := include "sumologic.sourceCategoryPrefix" . | quote }}
+{{- $sourceCategoryReplaceDash := include "sumologic.sourceCategoryReplaceDash" . | quote }}
+{{- $yamlFile | replace "exporters.zipkin.url_replace" $tracesEndpoint | replace "processors.source.name.replace" $sourceName | replace "processors.source.category.replace" $sourceCategory | replace "processors.source.category_prefix.replace" $sourceCategoryPrefix | replace "processors.source.category_replace_dash.replace" $sourceCategoryReplaceDash | nindent 2 }}
```

Comment:
I believe there is no need to use `_helpers` in this case
`replace "processors.source.name.replace" $sourceName` could be just `replace "processors.source.name.replace" .Values.sumologic.sourceName`